internal/controller/job.go

```diff
@@ -115,6 +116,18 @@ func initJob(workbench defaultv1alpha1.Workbench, config Config, uid string, app
 		appImage = fmt.Sprintf("%s/%s:%s", app.Image.Registry, app.Image.Repository, appVersion)
 	}
 
+	// Handle user with default value
+	user := workbench.Spec.Server.User
+	if user == "" {
+		user = "chorus"
+	}
+
+	// Handle UserID with default value
+	userID := workbench.Spec.Server.UserID
+	if userID == 0 {
+		userID = 1001
```

Comment:
Both values are already hardcoded in the image. https://github.com/CHORUS-TRE/images/pull/140

Make them optional with no default values so it picks whatever comes from the image.